src/bikeracks/templates/helpers.js

```diff
@@ -17,15 +17,10 @@ class BikeRack {
     setMarkerColor() {
         // based on upvote and downvote count
         let markerColor,
-            status = this.state.status;
-        
-        if (status === "approved") {
-            markerColor = approvedMarkerColor;
-        }
-        else if (status === "not_approved") {
-            markerColor = notApprovedMarkerColor;
-        }
-        
+            status = this.state.upvote_count > this.state.downvote_count;
+            
+        markerColor = status? approvedMarkerColor : notApprovedMarkerColor;
```

Comment:
Put a space between `status` and `?`. In JS it doesn't matter, but in some languages (e.g. ruby), `?` is a valid character in identifiers and it's convention to have boolean methods with names like `empty?`.

Reply:
Done.